classes/class-hmbkp-path.php

```diff
@@ -62,20 +64,17 @@ private function __wakeup() {}
 	/**
 	 * Returns the *Singleton* instance of this class.
 	 *
-	 * @staticvar HMBKP_PAth $instance The *Singleton* instances of this class.
+	 * @staticvar HMBKP_Path $instance The *Singleton* instances of this class.
 	 *
 	 * @return HMBKP_Path The *Singleton* instance.
 	 */
 	public static function get_instance() {
 
-		static $instance = null;
```

Comment:
@pdewouters `static` is a special case where it will only set it to null if it was never set before. However, I prefer the pattern of storing the static variable on the class, like so:

```PHP
class Foo {

    private static $instance;

    public static function get_instance() {
        if ( ! self::$instance ) {
            self::$instance = new HMBKP_Path();
        }
        return self::$instance;
    }
```